mdbenchmark/analyze.py

```diff
@@ -127,13 +135,25 @@ def analyze(directory, plot, ncores):
         'gromacs', 'nodes', 'ns/day', 'run time [min]', 'gpu', 'host', 'ncores'
     ])
 
+    errors = {'performance': [], 'ncores': []}
+
     for i, sim in enumerate(bundle):
-        df.loc[i] = analyze_run(sim)
+        df.loc[i], errors = analyze_run(sim, errors)
+
+    if errors['performance'] or errors['ncores']:
```

Comment:
Instead of checking for errors why not just check if you have `NaN` values in the dataframe? This also avoids having to add the `error` parameter to `analyze_run`.

Reply:
That's a good idea. I just changed missing values to `NaN` recently, after implementing the error collection. Getting rid of that is nicer.